source/server/getting-started/configuration/sponge-conf.rst

```diff
@@ -9,287 +9,472 @@ The global.conf file contains the global configuration settings for Sponge. This
 directory in your server folder. Many of these properties can be also overridden per-world or per-dimension type by
 using the config files in the subfolders of config/worlds.
 
-Below is a table with all available settings inside the global.conf file. Note that certain sections will not be filled
-immediately, and may optionally be added to the file when the server encounters them. There's also full example of a
-unmodified ``global.conf`` file at the bottom of this page, below the following table:
+Below is a list of available config settings and their comments inside the global.conf file. Note that certain sections
+will not be filled immediately, and may optionally be added to the file when the server encounters them. There's also
+full example of a unmodified ``global.conf`` file at the bottom of this page, below the following table:
```

Comment:
"... also a full example of *an* unmodified ..."
Also, it's a list now, not a table.